src/components/shared/VirtualList.js

```diff
@@ -172,8 +172,14 @@ type VirtualListProps = {|
   +disableOverscan: boolean,
   +columnCount: number,
   +containerWidth: CssPixels,
-  +role?: string,
+  // by using these props we are effectively telling screen readers that
+  // the browser (and our code) will take care of all accessibilty issues (like focus, navigation)
+  // aria-label is used when there's no visible label/name for the component
   +ariaLabel?: string,
+  // the container gets the role of "tree" and it's children are "tree items"
+  +role?: string,
```

Comment:
Looking at this now, I wonder if we shouldn't call it `ariaRole` (even if the HTML uses `role`). I think this would make more obvious the "role" (pun intended) of this property.